airbyte-integrations/connectors/source-looker/source_looker/components.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import dataclass
+from typing import Any, Mapping, Optional
+
+import pendulum
+import requests
+from airbyte_cdk.sources.declarative.auth.declarative_authenticator import NoAuth
+
+API_VERSION = "4.0"
+
+
+@dataclass
+class LookerAuthenticator(NoAuth):
```

Comment:
can you add a comment explaining how this work and/or how this is different from the existing session authenticator so we know what features are missing from the CDK?